src/content/en/2020/seo.md

```diff
@@ -83,493 +83,603 @@ Response Code | All Devices | Mobile | Desktop
+{# TODO(authors): Can you clarify how a link can be both outgoing and internal? #}
+The median page has 61 outgoing internal links on desktop and 54 on mobile. This is down 12.8% and 10% respectively from [last year's analysis](../2019/seo#linking). This suggests that sites are not maximizing the ability to improve the crawlability and link equity flow through their pages in the way they did the year before.
 
 {{ figure_markup(
   image="seo-outgoing-links-internal.png",
-  caption="Outgoing Links (Internal)",
+  caption="Distribution of the number of outgoing internal links per page.",
   description="Bar chart showing the number of internal outgoing links per percentile (10, 25, 50, 75, and 90). The median desktop page features 61 outgoing internal links while the median mobile page features 54 outgoing internal links.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTeddX0_5hUvxpYkd-927hEBlIqzuQiFn5450F2gNc9RJ5K8Wy0ln3KKD-gPWAxJ5Lo3H_km4ljHQEt/pubchart?oid=739265254&format=interactive",
   sheets_gid="775602646",
   sql_file="pages_wpt_bodies_by_device_and_percentile.sql"
-  )
-}}
+) }}
 
-The median homepage is linking to external sites 7 times on desktop and 6 times on mobile.
+{# TODO(authors): Add your interpretation of these results. What do you hope readers will get out of it? If there's not much to say, would anyone miss it if it was dropped? Also, please clarify the redundancy of outgoing and external for a non-SEO like myself :) #}
+The median home page is linking to external sites 7 times on desktop and 6 times on mobile.
 
 {{ figure_markup(
   image="seo-outgoing-links-external.png",
-  caption="Outgoing Links (External)",
-  description="Bar chart showing the number of external outgoing links per percentile (10, 25, 50, 75, and 90). The median homepage is linking to external sites 7 times on desktop and 6 times on mobile.",
+  caption="Distribution of the number of outgoing expernal links per page.",
+  description="Bar chart showing the number of external outgoing links per percentile (10, 25, 50, 75, and 90). The median home page is linking to external sites 7 times on desktop and 6 times on mobile.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTeddX0_5hUvxpYkd-927hEBlIqzuQiFn5450F2gNc9RJ5K8Wy0ln3KKD-gPWAxJ5Lo3H_km4ljHQEt/pubchart?oid=391564643&format=interactive",
   sheets_gid="775602646",
   sql_file="pages_wpt_bodies_by_device_and_percentile.sql"
-  )
-}}
+) }}
 
-#### Mobile vs. Desktop Links
+#### Mobile versus desktop links
 
 There is a disparity in the links between mobile and desktop that will negatively impact sites as Google becomes more committed to mobile-only indexing rather than just mobile-first indexing. This is illustrated in the 62 links on mobile versus the 68 links on desktop for the median web page.
 
+{# TODO(authors): What is a text link and how is that different from all the other links we've been talking about? At least to a non-SEO these all seem non-descript. #}
```

Comment:
@aleyda

Reply:
@rviscomi you can use a "text" as a anchor text for links, or you can use an "image" as an anchor text. This stat show the usage of "text based links" only. There's another one of "image based links". I just pushed a change since I saw the "image links" paragraph had been reworded in a way that made this much more difficult to understand (it had been changed from the original one that I think as an SEO was far easier to understand). Ideally links should be using "text" as anchor text, rather than "images", so it's important to differentiate.

Reply:
Thanks for clarifying. The "Text versus image links" section below is helpful context. Should this section be moved after that one so readers are primed to understand the distinction?

Reply:
Good call @rviscomi ! I have placed it before now :)